edxval/tests/test_api.py

```diff
@@ -687,3 +678,240 @@ def test_existing_video_in_destination_course_id(self):
 
         self.assertEqual(len(original_videos), 2)
         self.assertTrue(set(copied_videos) == set(original_videos))
+
+
+class ExportTest(TestCase):
+    """Tests export_to_xml"""
+    def setUp(self):
+        mobile_profile = Profile.objects.create(profile_name=constants.PROFILE_MOBILE)
+        desktop_profile = Profile.objects.create(profile_name=constants.PROFILE_DESKTOP)
+        Video.objects.create(**constants.VIDEO_DICT_STAR)
+        video = Video.objects.create(**constants.VIDEO_DICT_FISH)
+        EncodedVideo.objects.create(
+            video=video,
+            profile=mobile_profile,
+            **constants.ENCODED_VIDEO_DICT_MOBILE
+        )
+        EncodedVideo.objects.create(
+            video=video,
+            profile=desktop_profile,
+            **constants.ENCODED_VIDEO_DICT_DESKTOP
+        )
+
+    def assert_xml_equal(self, left, right):
+        """
+        Assert that the given XML fragments have the same attributes, text, and
+        (recursively) children
+        """
+        def get_child_tags(elem):
+            """Extract the list of tag names for children of elem"""
+            return [child.tag for child in elem]
+
+        for attr in ['tag', 'attrib', 'text', 'tail']:
```

Comment:
does this verify the other XML attributes: `'profile', 'url', 'file_size', 'nitrate'`?